_episodes/01-basics.md

```diff
@@ -1,13 +1,20 @@
 ---
-layout: page
-title: Testing
-subtitle: Basics of Testing
-minutes: 5
+title: Basics of Testing
+teaching: 5
+exercises: 0
+questions:
+- "Why would you not want to test?"
```

Comment:
Each pages starts with a question in the newer template that is hopefully answered by the end of the section. Quite likely not the best question to start with-.